lib/Cache.php

```diff
@@ -29,7 +29,7 @@ public function get(string $key): Promise;
      * @param $ttl int Timeout in seconds. The default `null` $ttl value indicates no timeout. Values less than 0 MUST
      * throw an \Error.
      *
-     * @return Promise Resolves either successfully or fails with a CacheException on failure.
+     * @return Promise<void> Resolves either successfully or fails with a CacheException on failure.
      */
     public function set(string $key, string $value, int $ttl = null): Promise;
```

Comment:
Actually I think that FileCache should be adjusted here since it can currently return int.
https://github.com/amphp/cache/blob/a90f3017bf20a606b02eebd600052b14cff3ecd4/lib/FileCache.php#L131

Reply:
Done in https://github.com/amphp/cache/pull/10